components/raftstore/src/store/fsm/apply.rs

```diff
@@ -234,16 +240,15 @@ pub enum ExecResult<S> {
     },
 }
 
-/// The possible returned value when applying logs.
 pub enum ApplyResult<S> {
     None,
-    Yield,
-    /// Additional result that needs to be sent back to raftstore.
-    Res(ExecResult<S>),
-    /// It is unable to apply the `CommitMerge` until the source peer
-    /// has applied to the required position and sets the atomic boolean
-    /// to true.
-    WaitMergeSource(Arc<AtomicU64>),
+    Res(Box<ExecResult<S>>),
```

Comment:
Is it necessary to use `Box` here?

Reply:
I just want to reduce cost of copy `ApplyResult`....